rooms.js

```diff
@@ -906,6 +941,8 @@ let BattleRoom = (() => {
 	BattleRoom.prototype.expireTimer = null;
 	BattleRoom.prototype.active = false;
 
+	BattleRoom.prototype.modlog = fs.createWriteStream(path.resolve(__dirname, 'logs/modlog/modlog_battle.txt'), {flags:'a+'});
```

Comment:
Wouldn't putting it in the constructor make a separate write stream for
every battleroom?